bento/src/main/java/com/yelp/android/bento/componentcontrollers/ListViewComponentController.kt

```diff
@@ -46,6 +47,18 @@ class ListViewComponentController(val listView: ListView) :
         listView.setTag(R.id.bento_list_component_controller, this)
     }
 
+    /**
+     * The scrolling capacity of a [ListView] depends on the ListView itself. To make a
+     * ListView non scrollable, you must subclass the ListView itself.
+     */
+    @Suppress("UNUSED_PARAMETER")
+    override var isScrollable: Boolean
+        get() = listView.canScrollHorizontally(-1)
+                || listView.canScrollHorizontally(1)
+                || listView.canScrollVertically(-1)
+                || listView.canScrollVertically(1)
+        set(value) = Unit
```

Comment:
Hmm.. for this could we set the touch listener on the ListView and consume all the DRAG actions if this value is toggled?

Reply:
So, touch listener seem to be inconsistent in how they work. Typically, it does not work for devices with keyboard, as they can always scroll.
One possibility could be to also intercept keyboard events?

Reply:
Also, apparently using the touch listener can be tricky: if one of the view inside the list view intercept the touch (like, a button), then the touch listener will be ignored, and a swipe will be possible.